src/components/editMeasure/populationCriteria/PopulationCriteriaHome.tsx

```diff
@@ -112,10 +111,11 @@ export function PopulationCriteriaHome() {
   const SupplementalElementsComponent = useMemo(
     () =>
       lazy(() => {
-        if (measure?.model === Model.QDM_5_6) {
-          return import("./supplementalData/qdm/QdmSupplementalElements");
-        } else {
-          return import("./supplementalData/qiCore/SupplementalElements");
+        if (
+          measure?.model === Model.QDM_5_6 ||
+          measure?.model === Model.QICORE
+        ) {
+          return import("./supplementalData/SupplementalData");
```

Comment:
does it make sense to remove the lazy loading now?

Reply:
I think we can remove it. we can add if we decide to support a new model.

Reply:
@adongare Ok thanks so much Ashok, will do.

Reply:
@adongare Removed.